crates/sparrow-runtime/src/execute.rs

```diff
@@ -18,16 +18,14 @@ use crate::execute::operation::OperationContext;
 use crate::s3::S3Helper;
 use crate::RuntimeOptions;
 
-mod avro_arrow;
+pub(crate) mod avro_arrow;
```

Comment:
What do we need from this module? Should it be in `sparrow-arrow` which already has an `avro` module?

Reply:
We convert the avro values into the arrow arrays. Yeah that's a good idea. I'll move that over